aries-site/src/examples/components/button/ButtonIconExample.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { Box, Button, ResponsiveContext } from 'grommet';
+import { FormPrevious, FormNext } from 'grommet-icons';
+
+export const ButtonIconExample = () => {
+  const size = React.useContext(ResponsiveContext);
+
+  return (
+    <Box
+      direction={size !== 'small' ? 'column' : 'row'}
+      gap={size !== 'small' ? 'small' : 'medium'}
+    >
+      <Box direction={size !== 'small' ? 'row' : 'column'} gap="medium">
```

Comment:
The buttons seem like they aren't respecting the default height (they're all growing to match the icon only size) -- maybe we need an `align="center"` when they are in a row? Also, wondering if Chris intended for them all to be in a single row across. That was my impression.

Reply:
I'll fix the height. Recommend we get the main stuff out the door, then can tweak individual examples as necessary. Okay with that approach?